packages/rum-core/src/domain/tracing/tracer.ts

```diff
@@ -122,10 +121,11 @@ function injectHeadersIfTracingAllowed(
     return
   }
   const traceId = createTraceIdentifier()
-  context.traceSampled =
-    !isNumber(configuration.traceSampleRate) || isTraceSampled(traceId, configuration.traceSampleRate)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is it okay to remove `!isNumber(configuration.traceSampleRate)`?

Reply:
Yes it is ok, because it is already done in [configuration.ts](https://github.com/DataDog/browser-sdk/blob/166489ddc05b31339571d57d12d7cbf298bb441a/packages/rum-core/src/domain/configuration/configuration.ts#L173) with `isSampleRate()` which check if it's a valid value for a sample rate or `undefined`.

And later we set the default value to `100` if it was `undefined` where before we would still keep it undefined and sample the trace when `!isNumber(undefined)`